lib/ops/unaryOps/unaryOpMinus.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+var BetalBlob = require("../../blob/betal");
+var NumberBlob = require("../../blob/number");
+var StringBlob = require("../../blob/string");
+var ListBlob = require("../../blob/list");
+var MapBlob = require("../../blob/map");
+var Constants = require("../../blob/constants");
+
+function unaryOpMinus(operand) {
+	
+	/*
+     * Base: http://www.ecma-international.org/ecma-262/5.1/#sec-11.4.7
+     */
+
+    let result = Constants.BETAL_NULL;
+
+    if(operand instanceof BetalBlob) {
+        result = new BetalBlob(operand._wrapped == "false")
```

Comment:
This comment is regarding the encapsulation issue. You can use `operand.isFalse()` instead of the check `operand._wrapped === "false"`. Again use triple equals when doing equality checks.